presto-blackhole/src/test/java/com/facebook/presto/plugin/blackhole/TestBlackHoleSmoke.java

```diff
@@ -187,6 +192,40 @@ public void fieldLength()
         assertThatQueryReturnsValue("DROP TABLE nation", true);
     }
 
+    @Test
+    public void pageProcessingDelay()
+            throws Exception
+    {
+        Session session = testSessionBuilder()
+                .setCatalog("blackhole")
+                .setSchema("default")
+                .build();
+
+        Duration pageProcessingDelay = Duration.valueOf("2s");
```

Comment:
This seems a bit long to delay a test. Could we make it shorter and still have a reasonable test?